openedx2zim/constants.py

```diff
@@ -20,6 +20,8 @@
 
 IMAGE_FORMATS = ["png", "jpeg", "jpg", "gif"]
 
+AUDIO_FORMATS = ["mp3", "ogg"]
```

Comment:
We should probably add more as browsers support more than this and it would be unfortunate to block it at our level. See nautilus JS file for a list.

Reply:
Added more formats